src/PAModel/CanvasDocument.cs

```diff
@@ -51,6 +51,7 @@ public class CanvasDocument
         internal TemplatesJson _templates;
         internal ThemesJson _themes;
         internal ResourcesJson _resourcesJson;
+        internal AppCheckerResultJson _appCheckerResultJson;
```

Comment:
Per #94, should we just keep .sarif it it's own \other\appcheckecer.sarif file? May get too large to put into entropy. 
And then the guidance is that  all of \other can get overwritten in a merge.

Reply:
Discussed offline, this is pretty much our best option for now.